CODEOWNERS

```diff
@@ -1 +1 @@
-*   @fstien @hj1611
+*   @fstien @hj1611 @aideencostello
```

Comment:
On the CODEOWNERS bit, I think it's best if we leave only @fstien as code owner, as he has the overview of the project, and whether something could break existing clients, he should get final approval of whether something is merged or not. Feel free to remove me from the list as well :)

Reply:
I think that would make it impossible for @fstien to merge PRs

Reply:
(I was hoping to be able to get involved in reviewing @fstien's PRs)

Reply:
Oh don't worry you can still review prs if you're not a code owner, and merging PRs should work fine through admin access :)